platform-mbean/src/main/java/org/jboss/as/platform/mbean/BufferPoolResourceDefinition.java

```diff
@@ -45,16 +45,16 @@
 class BufferPoolResourceDefinition extends SimpleResourceDefinition {
 
 
-    private static AttributeDefinition MEMORY_USED_NAME = SimpleAttributeDefinitionBuilder.create(PlatformMBeanConstants.MEMORY_USED_NAME, ModelType.LONG, false)
+    private static AttributeDefinition MEMORY_USED_NAME = SimpleAttributeDefinitionBuilder.create(PlatformMBeanConstants.MEMORY_USED_NAME, ModelType.LONG, true)
```

Comment:
Why are the platform mbean attributes being changed? I don't think there's a situation where these will be undefined.

java.lang.management.BufferPoolMXBean won't return null from any of its methods.